static/app/views/settings/components/teamSelect/teamSelectForMember.tsx

```diff
@@ -110,21 +111,27 @@ function TeamSelect({
 
   return (
     <Panel>
-      <PanelHeader hasButtons>
-        {t('Team')}
-        <DropdownAddTeam
-          disabled={disabled}
-          isLoadingTeams={isLoadingTeams}
-          isAddingTeamToMember
-          canCreateTeam={false}
-          onSearch={onSearch}
-          onSelect={onAddTeam}
-          onCreateTeam={onCreateTeam}
-          organization={organization}
-          selectedTeams={selectedTeams.map(tm => tm.slug)}
-          teams={teams}
-        />
-      </PanelHeader>
+      <TeamPanelHeader>
+        <div>{t('Team')}</div>
+        <div />
```

Comment:
whats this doin

Reply:
It's a CSS grid with 4 columns. This column doesn't need text on the header.